akka-http-jwt/src/main/resources/reference.conf

```diff
@@ -0,0 +1,33 @@
+#######################################
+# akka-http-jwt Reference Config File #
+#######################################
+
+# This is the reference config file that contains all the default settings.
+# Make your edits/overrides in your application.conf.
+
+akka.http.jwt {
+  # Enables or disables the JWT signature validation.
+  # This is useful for development and testing purposes
+  # where you can still assert presence of claims without validating the signature.
+  dev = off
+
+  # The realm to use in the WWW-Authenticate header when a token is missing or invalid.
+  realm = "akka-http-jwt"
+
+  # Allows configuration for the JWT secrets used to verify tokens.
+  # An example config would be:
```

Comment:
I think the original code had some note about order of secrets being significant? Good to add that here.